File: app/src/main/java/io/oversec/one/view/AboutActivity.java
package io.oversec.one.view;

import android.os.Bundle;
import androidx.activity.compose.setContent;
import androidx.appcompat.app.AppCompatActivity;
import android.view.MenuItem;

import io.oversec.one.iab.IabUtil;
import io.oversec.one.ui.screen.AboutScreen;
import io.oversec.one.ui.theme.OneTheme;

public class AboutActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContent {
            OneTheme {
                AboutScreen(
                    onBackPressed = () -> finish(),
                    isIabAvailable = IabUtil.getInstance(this).isIabAvailable()
                );
            }
        };
    }

    @Override
    public boolean onOptionsItemSelected(MenuItem item) {
        if (item.getItemId() == android.R.id.home) {
            finish();
            return true;
        }
        return super.onOptionsItemSelected(item);
    }
}

File: app/src/main/java/io/oversec/one/view/AboutActivity.kt
package io.oversec.one.view

import android.content.Context
import android.content.Intent
import android.os.Bundle
import androidx.activity.compose.setContent
import androidx.appcompat.app.AppCompatActivity
import io.oversec.one.iab.IabUtil
import io.oversec.one.ui.screen.AboutScreen
import io.oversec.one.ui.theme.OneTheme

class AboutActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val isIabAvailable = IabUtil.getInstance(this).isIabAvailable

        setContent {
            OneTheme {
                AboutScreen(
                    isIabAvailable = isIabAvailable,
                    onBackPressed = { finish() }
                )
            }
        }
    }

    companion object {
        @JvmStatic
        fun show(ctx: Context) {
            val i = Intent(ctx, AboutActivity::class.java)
            ctx.startActivity(i)
        }
    }
}